test/sample-test/requirements.in

```diff
@@ -9,4 +9,4 @@ google-api-python-client==1.7.8
 google-cloud-storage==1.17.0
 fire==0.2.1
 yamale==2.0
-tfx==0.21.2
+tfx==0.21.4
```

Comment:
You'll need to regenerate requirements.txt too

Reply:
I tried myself, it seems like pip-compile is failing b/c the recent kubernetes incompatible dependency problem zhitao cc'ed you.

Reply:
I think I've already fixed the Kubernetes problem in TFX.

This problem is different - for some reason this same `requirements.in` pins kubernetes package to a really old version. As far as I know, `requirements.in` should not pin versions like that - only `requirements.txt` should.

Reply:
I see. Thanks.

Reply:
@Ark-kun @numerology is there any action item for this comment? I ran the update-requirements.sh but there is nothing is changed.

Reply:
Please update/rebase to the latest master and run the improved script:
```
cd test/sample-test
cat ../../sdk/python/requirements.in requirements.in | ../../backend/update_requirements.sh google/cloud-sdk:279.0.0 >requirements.txt
```

Reply:
I've also updated the `requirements.in` to remove some unneeded packages and relax the dependencies (e.g. `kubernetes`). See https://github.com/kubeflow/pipelines/pull/3536

Reply:
@Ark-kun the command result in requirements.txt to be empty.

Reply:
> @Ark-kun the command result in requirements.txt to be empty.

Looks like the requirements of `tfx==0.21.4` are self-inconsistent. I've fixed this issues in the TFX repo, but it's not release dyet.

Reply:
I temporarily update the tfx version in requirements.txt file. If it passes E2E tests, we might go with the workaround to unblock the release.